core/src/main/scala/com/metamx/tranquility/security/SSLContextMaker.scala

```diff
@@ -0,0 +1,45 @@
+package com.metamx.tranquility.security
+
+import java.io.FileInputStream
+import java.io.IOException
+import java.security.{NoSuchAlgorithmException, KeyStoreException, KeyManagementException, KeyStore}
+import java.security.cert.CertificateException
+import javax.net.ssl.{TrustManagerFactory, SSLContext}
+
+import com.metamx.common.scala.Logging
+import com.metamx.tranquility.config.PropertiesBasedConfig
+
+object SSLContextMaker extends Logging
+{
+  def createSSLContextOption(generalConfig: PropertiesBasedConfig): Option[SSLContext] = {
+    if (!generalConfig.tlsEnable) {
+      log.info("TLS is not enabled, skipping SSLContext creation.")
+      return Option.empty[SSLContext]
```

Comment:
It's more idiomatic to return `None` here. And it's even more idiomatic to avoid `return` in favor of more functional-ish looking code (where the entire function is effectively a single expression).

Reply:
Changed this to avoid `return` and replaced this with `None`